### PR TITLE
Add stylelint-stylistic

### PR DESCRIPTION
**Pull request by elirasza**
See https://github.com/stylelint/stylelint/issues/6753#issuecomment-1493471819

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +115,8 @@
 - [stylelint-value-no-exposed-custom-properties](https://github.com/denisraslov/stylelint-value-no-exposed-custom-properties) - Disallow usage of exposed custom properties.
 - [stylelint-value-no-unknown-custom-properties](https://github.com/csstools/stylelint-value-no-unknown-custom-properties) - Disallow unknown custom properties.
 - [stylelint-z-index-value-constraint](https://github.com/kristerkari/stylelint-z-index-value-constraint) - Specify minimum and maximum constraint value for z-index.
+- [stylelint-stylistic](https://github.com/elirasza/stylelint-stylistic) - Enforce several stylistic rules.
```
Comment (ybiquitous) [inline README.md:118]: Thanks! Can you sort the plugins alphabetically?

Comment (XhmikosR) [reply README.md]: Done!

**Review by ybiquitous — APPROVED** [review]
Thanks, guys! 👍🏼

**Comment by XhmikosR** [comment]
Just wondering,mMaybe it should read "Enforce stylelistic rules deprecated in Stylelint v15"?

**Comment by ybiquitous** [comment]
I have no objection and will accept the change if @elirasza allows it.

**Comment by elirasza** [comment]
The change is OK for me !

**Pull request merged**